src/panels/config/scene/ha-scene-editor.ts

```diff
@@ -119,12 +115,23 @@ export class HaSceneEditor extends PreventUnsavedMixin(
 
   @state() private _devices: string[] = [];
 
-  @state()
+  @consume({ context: fullDevicesContext, subscribe: true })
   private _deviceRegistryEntries: DeviceRegistryEntry[] = [];
 
   @state()
+  @consume({ context: fullEntitiesContext, subscribe: true })
   private _entityRegistryEntries: EntityRegistryEntry[] = [];
 
+  @consume({ context: fullEntitiesContext, subscribe: true })
+  @transform<string | undefined, EntityRegistryEntry[]>({
+    transformer: function (this: HaSceneEditor, value) {
```

Comment:
```suggestion
    transformer: (this: HaSceneEditor, value: EntityRegistryEntry[]) => value.find(({ entity_id }) => entity_id === this._scene?.entity_id)
        ?.categories?.scene
```

Reply:
This is because we have this transformer `this` in here, when I rename it and use arrow function I get a type error :shrug: